scripts/upgrade_upstream.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Thanks for adding these, is this added through ci-mgmt?

Reply:
Not yet, since scripts are free-floating. Can elevate to Makefile targets at some point if useful.